### ISSUE TITLE
extern  static + wasm_bindgen incorrect error highlighting

### ISSUE THREAD

**Issue by pauldorehill** [intellij-rust/intellij-rust#4916]
## Environment

* **IntelliJ Rust plugin version:** 0.2.114.2151-193
* **Rust toolchain version:** 1.41.0 stable-x86_64-pc-windows-msvc
* **IDE name and version:**  CLion 2019.3.3
* **Operating system:** Windows 10 Pro 1909

## Problem description

The use of a `static` from an `extern`  + `#[wasm_bindgen]` code block incorrectly highlights the code with a red compiler error of:

`Use of extern static is unsafe and requires unsafe function or block [E0133]`

![image](https://user-images.githubusercontent.com/38935217/73699817-f1482800-4738-11ea-9fcf-4ab1206dc65a.png)

However `unsafe` is not required & also gives a warning with cargo check / clippy:
```
warning: unnecessary `unsafe` block
  --> src\lib.rs:78:13
   |
78 |     let a = unsafe { myT.log() };
   |             ^^^^^^ unnecessary `unsafe` block
   |
   = note: `#[warn(unused_unsafe)]` on by default
```

## Steps to reproduce
```rust
#[wasm_bindgen]
extern "C" {
    type MyT;

    static myT: MyT;

    #[wasm_bindgen(method)]
    fn log(this: &MyT);
}

#[wasm_bindgen]
pub fn test() {
    let a = myT.log();
}
```

**Comment by ortem** [comment]
Related to #3129
